community/modules/scheduler/schedmd-slurm-gcp-v6-controller/main.tf

```diff
@@ -40,6 +40,31 @@ locals {
 
 }
 
+locals {
+  synt_suffix       = substr(md5("${var.project_id}${var.deployment_name}"), 0, 5)
+  synth_bucket_name = "${local.slurm_cluster_name}${local.synt_suffix}"
+}
+
+module "bucket" {
+  source  = "terraform-google-modules/cloud-storage/google"
+  version = "~> 3.0"
```

Comment:
nit: elsewhere in toolkit we reference 3.4 and 4.0 as a min. Maybe pick similar for consistency.

Reply:
I'm sticking to the same version used in SchedMD module, would keep it this way until we rethink this whole situation (hopefully soon).